ipam/README.md

```diff
@@ -195,9 +213,16 @@ Class | Method | HTTP request | Description
  - [AddressBlock](docs/AddressBlock.md)
  - [AsmEnableBlock](docs/AsmEnableBlock.md)
  - [AsmGrowthBlock](docs/AsmGrowthBlock.md)
+ - [AssociateConfigProfileToObjectsRequest](docs/AssociateConfigProfileToObjectsRequest.md)
+ - [AssociateObjectToConfigProfilesRequest](docs/AssociateObjectToConfigProfilesRequest.md)
+ - [AssociatedHost](docs/AssociatedHost.md)
  - [BulkCopyError](docs/BulkCopyError.md)
  - [BulkCopyIPSpace](docs/BulkCopyIPSpace.md)
  - [BulkCopyIPSpaceResponse](docs/BulkCopyIPSpaceResponse.md)
+ - [BulkCreateMacAddressItemResponse](docs/BulkCreateMacAddressItemResponse.md)
+ - [BulkMacAddressItem](docs/BulkMacAddressItem.md)
+ - [CPSubnet](docs/CPSubnet.md)
```

Comment:
This name should be more user friendly I think

Reply:
Will fix it in a separate PR along with description for federated realms.